analysers/Analyser_Osmosis.py

```diff
@@ -45,7 +45,10 @@ class Analyser_Osmosis(Analyser):
     tags->'highway' LIKE '%_link' AS is_link,
     (tags?'junction' AND tags->'junction' = 'roundabout') AS is_roundabout,
     (tags?'oneway' AND tags->'oneway' IN ('yes', 'true', '1', '-1')) AS is_oneway,
-    (tags?'area' AND tags->'area' != 'no') AS is_area,
+    CASE
+      WHEN tags?'area' THEN tags->'area' != 'no'
+      ELSE (is_polygon AND tags->'highway' = 'pedestrian')
```

Comment:
Unlike the original, this can return NULL.
Maybe:
```suggestion
      ELSE (is_polygon AND tags?'highway' AND tags->'highway' = 'pedestrian')
```

Reply:
Everything without the tag `highway` is already filtered out in the WHERE statement, are you sure this is necessary to do it here in the SELECT again?

Reply:
Right!